src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericNumberInput.js

```diff
@@ -41,20 +34,20 @@ export const GenericNumberInput = ({
     disabled: !context.editMode,
     id: `number-input-${parameterData.id}`,
   };
+  const errorMessages = {
+    required: t('views.scenario.scenarioParametersValidationErrors.required', 'This field is required'),
+  };
```

Comment:
IIUC, the prop `errorMessages` no longer exists in `BasicNumberInput`, is this correct ?

Reply:
yes, you are right, I remove it :pray: